keeper/src/main/scala/zio/keeper/hyparview/HyParViewConfig.scala

```diff
@@ -17,19 +20,35 @@ object HyParViewConfig {
     concurrentIncomingConnections: Int
   ) {
 
-    val prettyPrint: String = s"""activeViewCapacity: $activeViewCapacity
-                                 |passiveViewCapacity: $passiveViewCapacity
-                                 |arwl: $arwl
-                                 |prwl: $prwl
-                                 |shuffleNActive: $shuffleNActive
-                                 |shuffleNPassive: $shuffleNPassive
-                                 |shuffleTTL: $shuffleTTL
-                                 |connectionBuffer: $connectionBuffer
-                                 |userMessagesBuffer: $userMessagesBuffer
-                                 |concurrentIncomingConnections: $concurrentIncomingConnections""".stripMargin
+    val prettyPrint: String =
+      s"""address: $address
+         |activeViewCapacity: $activeViewCapacity
+         |passiveViewCapacity: $passiveViewCapacity
+         |arwl: $arwl
+         |prwl: $prwl
+         |shuffleNActive: $shuffleNActive
+         |shuffleNPassive: $shuffleNPassive
+         |shuffleTTL: $shuffleTTL
+         |connectionBuffer: $connectionBuffer
+         |userMessagesBuffer: $userMessagesBuffer
+         |concurrentIncomingConnections: $concurrentIncomingConnections""".stripMargin
   }
 
+  trait Service {
+    val getConfigSTM: STM[Nothing, Config]
+
+    val getConfig: UIO[Config] =
```

Comment:
Does it make sense to expose this?

Reply:
Removed. Just the function in the object is enough